crypto/crypto_aes.c

```diff
@@ -57,7 +59,7 @@ useaesni(void)
 	static int aesnigood = -1;
 	uint8_t key[32];
 	uint8_t ptext[16];
-	size_t i;
+	uint8_t i;
```

Comment:
So, the idea here is that a "straightforward" reading of the code should match what it actually does.  `i` is an array index, so it should be a `size_t`; where we set a byte equal to `i` or `0x11 * i` what we really mean is "set this byte to the lowest byte of..." and `& 0xff` is a way of making that explicit rather than simply relying on the implicit type conversion.